File: api-crypto/crypto/csharp/src/exceptions/AlreadyInitializedException.cs
using System;
using tr.gov.tubitak.uekae.esya.api.common.bundle;
/**
 * <p>Title: CC</p>
 * <p>Description: </p>
 * <p>Copyright: Copyright (c) 2002</p>
 * <p>Company: TUBITAK/UEKAE</p>
 * @author M. Serdar SORAN
 * @version 1.0
 */

//todo Annotation!
//@ApiClass
namespace tr.gov.tubitak.uekae.esya.api.crypto.exceptions
{
    public class AlreadyInitializedException : CryptoException
    {
        public AlreadyInitializedException() : base(Resource.message(Resource.INIT_EDILMIS)) { }
        public AlreadyInitializedException(Exception ex) : base(Resource.message(Resource.INIT_EDILMIS), ex) { }
        public AlreadyInitializedException(String mesaj) : base(mesaj) { }
    }
}

File: api-crypto/crypto/src/tr/gov/tubitak/uekae/esya/api/crypto/exceptions/AlreadyInitializedException.java
package tr.gov.tubitak.uekae.esya.api.crypto.exceptions;

import tr.gov.tubitak.uekae.esya.api.common.bundle.GenelDil;



/**
 * <p>Title: CC</p>
 * <p>Description: </p>
 * <p>Copyright: Copyright (c) 2002</p>
 * <p>Company: TUBITAK/UEKAE</p>
 * @author M. Serdar SORAN
 * @version 1.0
 */

public class AlreadyInitializedException
    extends CryptoException
{

     public AlreadyInitializedException()
     {
          super(GenelDil.mesaj(GenelDil.INIT_EDILMIS)); // Daha once initialize edilmis
     }


     public AlreadyInitializedException(Throwable ex)
     {
          super(GenelDil.mesaj(GenelDil.INIT_EDILMIS), ex); // Daha once initialize edilmis
     }


     public AlreadyInitializedException(String mesaj)
     {
          super(mesaj);
     }

}